packages/perseus/src/widgets/measurer/measurer.tsx

```diff
@@ -1,3 +1,7 @@
+import {
+    measurerLogic,
+    type PerseusMeasurerWidgetOptions,
```

Comment:
I'm surprised the linter isn't complaining about this. I thought we had an ESLint rule to group `type` imports together  in a separate group (at the bottom).

Reply:
Yeah, this is just how my IDE does it. Should I file a ticket to make our linter try and auto-fix these?